ipc/src/main/java/io/scalecube/ipc/netty/NettyBootstrapFactory.java

```diff
@@ -0,0 +1,74 @@
+package io.scalecube.ipc.netty;
+
+import io.netty.bootstrap.Bootstrap;
+import io.netty.bootstrap.ServerBootstrap;
+import io.netty.channel.ChannelOption;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.epoll.Epoll;
+import io.netty.channel.epoll.EpollEventLoopGroup;
+import io.netty.channel.epoll.EpollServerSocketChannel;
+import io.netty.channel.epoll.EpollSocketChannel;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.ServerSocketChannel;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.channel.socket.nio.NioServerSocketChannel;
+import io.netty.channel.socket.nio.NioSocketChannel;
+import io.netty.util.concurrent.DefaultThreadFactory;
+
+import java.util.concurrent.ThreadFactory;
+
+public final class NettyBootstrapFactory {
```

Comment:
Consider an opportunity to unify it with BoostrapFactory at transport module (not this PR)

Reply:
I was thinking to have separate one, not to depend on `scalecube-transport` in new modules: `gateway` and `ipc`.

Reply:
I was not speaking about adding this dependency, but more like to move this stuff into `utils` or some other common module.